### PR TITLE
Cookbook - relaxedtree classifier

### PR DESCRIPTION
**Pull request by OXPHOS**

### CODE REVIEW

File: doc/cookbook/source/examples/classifier/relaxedtree.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
+=======================
+Relaxed Tree Classifier
+=======================
+
+Relaxed Tree Algorithm, or Relaxed Hierarchy Algorithm, solves multi-class classification problems by exploiting the relaxed hierarchy structure of the data,
```
Comment (karlnapf) [inline doc/cookbook/source/examples/classifier/relaxedtree.rst]: The sentence doesnt make sense to from after "by exploiting"

File: doc/cookbook/source/references.bib [hunk 2]
```diff
@@ -59,4 +59,13 @@ @book{scholkopf2002learning
   author={B. Sch{\"o}lkopf and A.J. Smola},
   year={2002},
   publisher={MIT press}
-}
\ No newline at end of file
+}
+
+@inproceedings{gao2011discriminative,
```
Comment (karlnapf) [inline doc/cookbook/source/references.bib]: This is coming from where?

File: doc/cookbook/source/examples/classifier/relaxedtree.rst [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+-------
+Example
+-------
+
+Imagine we have files with training and test data. We create CDenseFeatures (here 64 bit floats aka RealFeatures) and :sgclass:`CMulticlassLabels` as
+
+.. sgexample:: relaxedtree.sg:create_features
+
+In order to run :sgclass:`CRelaxedTree`, we need to set the machine for confusion matrix and choose the kernel.
```
Comment (karlnapf) [inline doc/cookbook/source/examples/classifier/relaxedtree.rst]: This should be explained/ellaborated a bit more on

**Comment by karlnapf** [comment]
I currently cannot look at the rendered html
@vigsterkr it says "service not available" behind the preview build link

File: doc/cookbook/source/examples/classifier/relaxedtree.rst [hunk 4]
```diff
@@ -0,0 +1,41 @@
+=======================
```
Comment (karlnapf) [inline doc/cookbook/source/examples/classifier/relaxedtree.rst]: Please remove Classifier in the title (It is listed under classifier)
Rename to relaxed_tree.tst

File: doc/cookbook/source/references.bib (hunk 2 above)
Comment (OXPHOS) [inline doc/cookbook/source/references.bib]: you mean the publication or the format..? 
The format is from the BibTeX citing of google scholar..
and publication is in relaxedtree.h

Comment (karlnapf) [inline doc/cookbook/source/references.bib]: publication. Just wanted to know, all good then

File: doc/cookbook/source/references.bib [hunk 5]
```diff
@@ -59,4 +59,13 @@ @book{scholkopf2002learning
...
+  title={Discriminative learning of relaxed hierarchy for large-scale visual recognition},
+  author={T. Gao and D. Koller},
+  booktitle={Computer Vision (ICCV), 2011 IEEE International Conference on},
```
Comment (karlnapf) [inline doc/cookbook/source/references.bib]: the booktiitle seems a bit weirdly formatted though :)

**Comment by OXPHOS** [comment]
@karlnapf updated. not very satisfied though..

File: doc/cookbook/source/examples/classifier/relaxed_tree.rst [hunk 6]
```diff
@@ -0,0 +1,44 @@
+============
+Relaxed Tree
+============
+
+Relaxed tree algorithm, or relaxed hierarchy algorithm, solves multi-class classification problems by exploiting the relaxed hierarchy structure of the data,
```
Comment (karlnapf) [inline doc/cookbook/source/examples/classifier/relaxed_tree.rst]: The

**Comment by karlnapf** [comment]
Why not satisfied?
From my side, all this needs is data and a squash, and then we can merge it! :)

**Comment by OXPHOS** [comment]
@karlnapf data version updated and ready

**Comment by karlnapf** [comment]
Thanks! :)

**Pull request merged**